core/domain/rights_manager.py

```diff
@@ -873,7 +907,10 @@ def _deassign_role(committer, removed_user_id, activity_id, activity_type):
     committer_id = committer.user_id
     activity_rights = _get_activity_rights(activity_type, activity_id)
 
-    if not check_can_modify_activity_roles(committer, activity_rights):
+    user_can_modify_activity_roles = check_can_modify_voiceartist_in_activity(
```

Comment:
Will it allow voiceover admin to remove any role?

Reply:
Done